src/webui/www/private/preferences_content.html

```diff
@@ -1099,6 +1372,54 @@
         return ret;
     };
 
+    // Advanced Tab
+    const updateNetworkInterfaces = function(default_iface) {
```

Comment:
Opening settings multiple times w/o reloading throws a syntax error

`SyntaxError: redeclaration of const updateNetworkInterfaces`

I think `updateInterfaceAddresses` will as well.

Reply:
Yes. It seems that we have to use `var`, because the objects are still remained in browser after closing the dialog. Both `let` and `const` are not allowed redeclaration.